deploy/helm/sumologic/upgrade-1.0.0.sh

```diff
@@ -38,17 +40,359 @@ function check_required_command() {
 }
 
 check_required_command yq
+yq --version | grep 3.2.1 >/dev/null 2>&1 || { echo >&2 "yq version is invalid. It should be exactly 3.2.1. Please install it properly: https://github.com/mikefarah/yq/releases/tag/3.2.1"; exit 1; }
 check_required_command grep
 check_required_command sed
 check_required_command curl
 
 readonly OLD_VALUES_YAML="$1"
 readonly HELM_RELEASE_NAME="${2:-collection}"
 readonly NAMESPACE="${3:-sumologic}"
+readonly PREVIOUS_VERSION=0.17.2
 
 URL=https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/helm/sumologic/values.yaml
 curl -s $URL > new.yaml
 
+readonly KEY_MAPPINGS="
+eventsDeployment.nodeSelector:fluentd.events.statefulset.nodeSelector
+eventsDeployment.resources.limits.cpu:fluentd.events.statefulset.resources.limits.cpu
+eventsDeployment.resources.limits.memory:fluentd.events.statefulset.resources.limits.memory
+eventsDeployment.resources.requests.cpu:fluentd.events.statefulset.resources.requests.cpu
+eventsDeployment.resources.requests.memory:fluentd.events.statefulset.resources.requests.memory
+eventsDeployment.tolerations:fluentd.events.statefulset.tolerations
+sumologic.addStream:problems.sumologic.addStream
+sumologic.addTime:problems.sumologic.addTime
+sumologic.addTimestamp:fluentd.logs.output.addTimestamp
+sumologic.chunkLimitSize:fluentd.buffer.chunkLimitSize
+sumologic.eventCollectionEnabled:fluentd.events.enabled
+sumologic.events.sourceCategory:fluentd.events.sourceCategory
+sumologic.excludeContainerRegex:fluentd.logs.containers.excludeContainerRegex
+sumologic.excludeHostRegex:fluentd.logs.containers.excludeHostRegex
+sumologic.excludeNamespaceRegex:fluentd.logs.containers.excludeNamespaceRegex
+sumologic.excludePodRegex:fluentd.logs.containers.excludePodRegex
+sumologic.fluentd.buffer:fluentd.buffer.type
+sumologic.fluentd.buffer:problems.sumologic.fluentd.buffer
+sumologic.fluentdLogLevel:fluentd.logLevel
+sumologic.flushInterval:fluentd.buffer.flushInterval
+sumologic.k8sMesumologic.k8sMesumologic.k8sMesumologic.k8sMesumologic.k8sMesusMetadataFilter.cacheTtl:fluentd.metadata.cacheTtl
```

Comment:
please fix that

Reply:
seems like corrupted line 80, so going to remove that